lib/ShareProvider.js

```diff
@@ -344,7 +350,7 @@ ShareProvider.prototype = {
       }
     };
 
-    CustomizableUI.createWidget(shareButton);
+    widget = CustomizableUI.createWidget(shareButton);
```

Comment:
ah... it's actually declared at the top of the `_createButton()` function -> https://github.com/rlr/activity-streams/blob/2ff6007bcf8c77e44b9a73da56ee5f4586b8a50f/lib/ShareProvider.js#L231

Then re-set at the button after creating the widget.